src/hooks/tests/useBreadcrumbRoot.test.tsx

```diff
@@ -41,12 +50,11 @@ describe("useBreadcrumbRoot", () => {
     expect(result.current.text).toBe("John Doe's Patches");
   });
 
-  it("returns the correct breadcrumb root when the version is a commit", async () => {
-    const { result, waitForNextUpdate } = renderHook(
+  it("returns the correct breadcrumb root when the version is a commit", () => {
+    const { result } = renderHook(
       () => useBreadcrumbRoot(false, "admin", "spruce"),
       { wrapper: SameUserProvider }
     );
-    await waitForNextUpdate();
```

Comment:
It should make sense that we don't have to wait for an update if the version is a commit, because we skip the query in this case.